app/assets/stylesheets/application.scss

```diff
@@ -75,3 +75,73 @@ footer {
     margin: 0 0 18px 0;
   }
 }
+
+.hero-modified-body {
+  padding-top: 0px;
+  margin-top: -100px;
+}
+
+.container {
+  padding-top: 24px;
+}
+
+h2 {
+  padding-top: 24px;
+  padding-bottom: 24px;
+}
+
+// Overwrite Bulma's stupid h tag rules
+h1 {
```

Comment:
You said you wrote some Javascript for this, where is that?

Reply:
Also, since we are using Sass here, we can use inheritance to make the `<hx>` classes look a little DRYer.

Reply:
I axed the js

Reply:
also, I'm not sure what you are talking about with the css